Cartfile.resolved

```diff
@@ -1,5 +1,7 @@
 github "Quick/Nimble" "v5.1.1"
 github "Quick/Quick" "v1.1.0"
-github "ReactiveCocoa/ReactiveCocoa" "5.0.1"
-github "ReactiveCocoa/ReactiveSwift" "1.1.0"
+github "ReactiveCocoa/ReactiveCocoa" "5.0.0-alpha.5"
```

Comment:
Yes we should, and we will. We are making some final corrections to make the new release. But for the semantics of `~>` it will download the new version, and so use the new RAC's version. Maybe I should stall this until the new Core's release is made.